api/tests/core/mails/transactional/bookings/booking_confirmation_to_beneficiary_test.py

```diff
@@ -201,6 +202,36 @@ def test_should_return_offer_tags():
     assert email_data == expected
 
 
+@freeze_time("2021-10-15 12:48:00")
+def test_should_return_offer_features():
+    stock = offers_factories.ThingStockFactory(
+        features=["Feature 1", "Feature 2"],
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Mettre une vrai valeur comme en prod

```suggestion
        features=["VO", "IMAX"],